packages/main/src/types/SegmentedButtonSelectionMode.ts

```diff
@@ -1,19 +1,19 @@
 /**
- * Different SegmentedButton modes.
+ * Different SegmentedButton selection modes.
  * @public
  */
-enum SegmentedButtonMode {
+enum SegmentedButtonSelectionMode {
 	/**
 	 * There is always one selected. Selecting one deselects the previous one.
 	 * @public
 	 */
-	SingleSelect = "SingleSelect",
+	Single = "Single",
 
 	/**
 	 * Multiple items can be selected at a time. All items can be deselected.
 	 * @public
 	 */
-	MultiSelect = "MultiSelect",
+	Multi = "Multi",
```

Comment:
Shouldn't this be "Multiple" as in the `ui5-calendar` component?

Reply:
Yes, it should.